docs/contributing.md

```diff
@@ -0,0 +1,122 @@
+# Contributing
+
+Contributions to Student Robitics' Runbook should follow Student Robitics'
+[Code of Conduct][code-of conduct] and made under a compatible [license][license].
+
+You may also find the [Zen of Python][pep-20] provides useful guidance.
+
+[code-of conduct]: https://opsmanual.studentrobotics.org/about-the-charity/code-of-conduct
+[license]: ./license.md
+[pep-20]: https://www.python.org/dev/peps/pep-0020/
+
+## Making big changes
+
+As the runbook is a resource which expects to be read by a large number of
+people, its current shape & content form a public API of sorts -- the way to
+navigate it as well as the locations of much of the content are likely
+muscle-memory to many contributors and readers. Making big changes to it
+therefore needs careful consideration to ensure that their workflows are not
+disrupted unduly.
+
+It's a good idea therefore to raise any large changes as a [new issue][new-issue]
+before commiting to them.
+
+Alternatively, if your change is more easily explained as a demonstration of the
+outcome, it's fine to raise a pull request showing that. Note however that your
+suggestion will still be discsused and may not be accepted, so you may wish to
```

Comment:
I think that we should await the resolution of ownership and control of the runbook before finalising this section. Please see the email sent on 14/02

Reply:
This PR aims to record the current state -- i.e: the original vision for the runbook.